cli/command/system/version.go

```diff
@@ -17,7 +16,7 @@ import (
 
 var versionTemplate = `Client:
  Version:      {{.Client.Version}}
- API version:  {{.Client.APIVersion}}
+ API version:  {{.Client.ActualAPIVersion}} (default version {{.Client.DefaultAPIVersion}})
```

Comment:
We should keep the message the same here ("downgraded from").

Can we make make the actual comparison in this template? i.e.; if actual != default, then print the "downgraded from .." part? I'm not too good in Go templates, but I think it should be possible